os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -280,6 +280,7 @@ g_mode = RTK_WIFI_NONE;
 		}
 
 		ret = cmd_wifi_on(RTK_WIFI_STATION_IF);
+		wifi_monitor_start_timer();
 		if (ret != RTK_STATUS_SUCCESS) {
 			ndbg("[RTK] Failed to start STA mode\n");
 			return wuret;

```

Comment:
Is it okay not to stop the timer?
So I think we'd better start the timer after checking the return of `cmd_wifi_on()`.

Reply:
@jeongchanKim If the wifi_monitor_stop_timer() is not called to delete the timer entry before cmd_wifi_off(), we will get the below warning log
"Deinitializing WIFI ...Need to delete 1 timer_entry" .
Also when cmd_wifi_on() called again, timer cannot be started with the below message 
"[timer_wrapper] Fail to find the timer_entry in timer table"

Reply:
or do you suggest that we will delete the timer based on the result of  cmd_wifi_on() instead of having separate function as stop timer?

Reply:
if cmd_wifi_on is fail, do we need start timer here?

Reply:
No, I'll move the start timer after it's checked the status of cmd_wifi_on like below
`ret = cmd_wifi_on(RTK_WIFI_STATION_IF);
if (ret != RTK_STATUS_SUCCESS) {
	ndbg("[RTK] Failed to start STA mode\n");
	return wuret;
}
wifi_monitor_start_timer();`

Reply:
Okay. Thanks.

Reply:
But do we need to have stop timer or will check the wifi_on condition to delete the timer before the start of next round?

Reply:
If you move the start timer after checking the wifi_on, then we will not start the timer when wifi_on fails.
Then there is no case timer is started regardless of wifi_on.
So I think we don't need to stop the timer for next round.